.github/workflows/release.yml

```
@@ -77,21 +95,241 @@ jobs:
+          mkdir -p /tmp/verify
+          cd /tmp/verify
+          gh release download "${TAG}" \
+            --repo "${{ github.repository }}" \
+            --pattern '*.tgz' \
+            --pattern '*.sig' \
+            --pattern '*.cert' \
+            --pattern 'provenance.intoto.jsonl'
+          ls -la
+
+      - name: Verify cosign signature
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:2:11: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)